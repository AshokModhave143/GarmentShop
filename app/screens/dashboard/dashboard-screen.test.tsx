import React from 'react'
import { DashboardScreen } from '..'
import { render } from '../../../test/test-util'

describe('Dashboard screen', () => {
  it('should render component without crashing', () => {
    const { toJSON } = render(<DashboardScreen />, {})
    expect(toJSON()).toMatchSnapshot()
  })
})
