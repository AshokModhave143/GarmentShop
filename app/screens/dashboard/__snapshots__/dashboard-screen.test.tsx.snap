// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard screen should render component without crashing 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
    testID="DashboardScreen"
  >
    <View
      onLayout={[Function]}
      style={
        Array [
          Array [
            Object {
              "backgroundColor": "#ffffff",
              "flex": 1,
              "height": "100%",
            },
            Object {
              "backgroundColor": "rgba(0, 0, 0, 0)",
            },
          ],
          Object {
            "paddingBottom": 0,
          },
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "backgroundColor": "#ffffff",
              "flex": 1,
              "height": "100%",
            },
            Object {
              "backgroundColor": "rgba(0, 0, 0, 0)",
            },
            Object {
              "paddingTop": 0,
            },
          ]
        }
      >
        <RCTScrollView
          contentContainerStyle={
            Array [
              Object {
                "alignItems": "stretch",
                "justifyContent": "flex-start",
              },
              Object {
                "backgroundColor": "rgba(0, 0, 0, 0)",
                "paddingHorizontal": 16,
              },
            ]
          }
          keyboardShouldPersistTaps="handled"
          style={
            Array [
              Object {
                "backgroundColor": "#ffffff",
                "flex": 1,
                "height": "100%",
              },
              Object {
                "backgroundColor": "rgba(0, 0, 0, 0)",
              },
            ]
          }
        >
          <View>
            <View
              style={
                Array [
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "justifyContent": "flex-start",
                    "paddingBottom": 24,
                    "paddingHorizontal": 16,
                    "paddingTop": 24,
                  },
                  Object {
                    "paddingBottom": 23,
                    "paddingHorizontal": 0,
                    "paddingTop": 12,
                  },
                ]
              }
            >
              <View
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "flex-start",
                    "borderRadius": 4,
                    "justifyContent": "center",
                    "opacity": 1,
                    "paddingHorizontal": 0,
                    "paddingVertical": 0,
                  }
                }
              >
                <View>
                  <RCTImageView
                    internal_analyticTag={null}
                    resizeMode="contain"
                    source={
                      Array [
                        Object {
                          "testUri": "../../../app/components/built-in/icon/icons/arrow-left.png",
                        },
                      ]
                    }
                    style={
                      Object {
                        "height": 100,
                        "overflow": "hidden",
                        "resizeMode": "contain",
                        "width": 100,
                      }
                    }
                  />
                </View>
              </View>
              <View
                style={
                  Object {
                    "flex": 1,
                    "justifyContent": "center",
                  }
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "color": "#ffffff",
                        "fontFamily": "Roboto-Regular",
                        "fontSize": 15,
                      },
                      Array [
                        Object {
                          "textAlign": "center",
                        },
                        Object {
                          "fontSize": 12,
                          "fontWeight": "bold",
                          "letterSpacing": 1.5,
                          "lineHeight": 15,
                          "textAlign": "center",
                        },
                      ],
                    ]
                  }
                >
                  dashboardScreen.howTo
                </Text>
              </View>
              <View
                style={
                  Object {
                    "width": 32,
                  }
                }
              />
            </View>
            <Text
              style={
                Array [
                  Object {
                    "color": "#ffffff",
                    "fontFamily": "Roboto-Regular",
                    "fontSize": 24,
                    "fontWeight": "bold",
                  },
                  Object {
                    "fontSize": 28,
                    "fontWeight": "bold",
                    "lineHeight": 38,
                    "marginBottom": 24,
                    "textAlign": "center",
                  },
                ]
              }
            >
              dashboardScreen.title
            </Text>
            <Text
              style={
                Array [
                  Object {
                    "color": "#ffffff",
                    "fontFamily": "Roboto-Regular",
                    "fontSize": 24,
                    "fontWeight": "bold",
                  },
                  Object {
                    "fontSize": 16,
                    "fontWeight": "normal",
                    "textAlign": "center",
                  },
                ]
              }
            >
              dashboardScreen.tagLine
            </Text>
          </View>
        </RCTScrollView>
      </View>
    </View>
  </View>
</RNCSafeAreaProvider>
`;
